src/libraries/System.Text.RegularExpressions/tests/FunctionalTests/Regex.Match.Tests.cs

```diff
@@ -84,11 +84,36 @@ static IEnumerable<(string Pattern, string Input, RegexOptions Options, int Begi
 
                 if (!RegexHelpers.IsNonBacktracking(engine))
                 {
-                    // Zero-width negative lookahead assertion: Actual - "abc(?!XXX)\\w+"
+                    // Zero-width negative lookahead assertion
                     yield return (@"abc(?!XXX)\w+", "abcXXXdef", RegexOptions.None, 0, 9, false, string.Empty);
 
-                    // Zero-width positive lookbehind assertion: Actual - "(\\w){6}(?<=XXX)def"
+                    // Zero-width positive lookbehind assertion
```

Comment:
NIT: Comment says positive lookbehind but there seems to be also negative ones. Of course don't reset CI just for this.

Reply:
Ok, I'll fix the comment separately. Thanks.